### PR TITLE
Added flip video. 

### PR DESCRIPTION
**Pull request by luiserdef**
From issue #10

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 let streaming = false;
 let video = null;
+let mirrorCamera=null;
+let flipVideo=false;
```
Comment (sbimochan) [inline script.js]:
```suggestion
let mirrorCamera = null;
let flipVideo = false;
```

File: script.js [hunk 2]
```diff
@@ -43,6 +45,8 @@ function swapSpinnerWithVideo() {
 function startup() {
   video = document.getElementById('video');
   canvas = document.getElementById('canvas');
+  mirrorCamera= document.querySelector('.mirror-camera');
```
Comment (sbimochan) [inline script.js]:
```suggestion
  mirrorCamera = document.querySelector('.mirror-camera');
```

File: script.js [hunk 3]
```diff
@@ -43,6 +45,8 @@ function swapSpinnerWithVideo() {
...
+  mirrorCamera.addEventListener('click',checkMirrorCamera)
```
Comment (sbimochan) [inline script.js]:
```suggestion
  mirrorCamera.addEventListener('click', checkMirrorCamera)
```

File: script.js [hunk 4]
```diff
@@ -70,6 +74,7 @@ function startup() {
         if (isNaN(resultHeight)) {
           resultHeight = width / (4 / 3);
         } else {
+          mirrorCamera.style.display="block";
```
Comment (sbimochan) [inline script.js]:
```suggestion
          mirrorCamera.style.display = "block";
```

File: script.js [hunk 5]
```diff
@@ -85,3 +90,15 @@ function startup() {
...
+  if(!flipVideo){
+    video.classList.add("flip-video");
+    mirrorCamera.style.background="#a53e5b";
+    flipVideo=true;
+  }else{
+    video.classList.remove("flip-video");
+    mirrorCamera.style.background="#a01f43";
+    flipVideo=false;
+  }
+}
```
Comment (sbimochan) [inline script.js]:
```suggestion
}

```

File: main.css [hunk 6]
```diff
@@ -97,3 +97,20 @@ body {
     height: 100%;
   }
 }
```
Comment (sbimochan) [inline main.css:105]:
```suggestion
}

```

File: main.css [hunk 7]
```diff
@@ -97,3 +97,20 @@ body {
...
+  padding: 5px;
+  cursor:pointer;
+  display:none;
+}
+
+.flip-video{
+    transform: rotateY(180deg);
+    -moz-transform:rotateY(180deg);
+    -webkit-transform:rotateY(180deg);    
+}
```
Comment (sbimochan) [inline main.css]:
```suggestion
}

```

**Review by sbimochan — CHANGES_REQUESTED** [review]
commit those suggestions first. Thanks

**Comment by sbimochan** [comment]
Make default flipped. and on clicking that flip button,let it be as now.

**Comment by sbimochan** [comment]
Change that button to rotate icon. Add some box-shadow so it looks more like a button. place it below video-player on bottom right. Or any other better UI is welcome.

File: index.html [hunk 8]
```diff
@@ -13,6 +13,7 @@
   </head>
   <body>
     <div class="video-window-wrapper">
+      <div class="mirror-camera">Flip camera</div>
```
Comment (sbimochan) [inline index.html]: It should be `<button>`

File: script.js [hunk 9]
```diff
@@ -85,3 +90,15 @@ function startup() {
     false
   );
 }
+
+function checkMirrorCamera(){
+  if(!flipVideo){
+    video.classList.add("flip-video");
+    mirrorCamera.style.background="#a53e5b";
+    flipVideo=true;
+  }else{
```
Comment (sbimochan) [inline script.js]:
```suggestion
  }else {
```

**Review by luiserdef — COMMENTED** [review]
I made some changes, to add the suggestions. #12

**Comment by sbimochan** [comment]
Nice.

**Pull request merged**